types/aws-lambda/test/lex-v2-tests.ts

```diff
@@ -117,3 +125,123 @@ const handler: LexV2Handler = async (event, context, callback) => {
+                            "interpretedValue": "7"
+                        }
+                    }
+                },
+                "confirmationState": "None",
+                "name": "IntentName",
+                "state": "InProgress"
+            },
+            "nluConfidence": 1
+        },
+        {
+            "intent": {
+                "slots": {},
+                "confirmationState": "None",
+                "name": "FallbackIntent",
+                "state": "InProgress"
+            }
+        }
+    ],
+    "proposedNextState": {
+        "intent": {
+            "slots": {
+                "Slot1": null,
+                "Slot2": null,
+                "Slot3": {
+                    "shape": "Scalar",
+                    "value": {
+                        "originalValue": "7",
+                        "resolvedValues": [
+                            "7"
+                        ],
+                        "interpretedValue": "7"
+                    }
+                }
+            },
+            "confirmationState": "None",
+            "name": "IntentName",
+            "state": "InProgress"
+        },
+        "dialogAction": {
+            "slotToElicit": "Slot1",
+            "type": "ElicitSlot"
+        }
+    },
+    "responseContentType": "text/plain; charset=utf-8",
+    "invocationSource": "DialogCodeHook",
+    "messageVersion": "1.0",
+    "transcriptions": [
+        {
+            "resolvedSlots": {
+                "Slot3": {
+                    "shape": "Scalar",
+                    "value": {
+                        "originalValue": "7",
+                        "resolvedValues": [
+                            "7"
+                        ]
+                    }
+                }
+            },
+            "resolvedContext": {
+                "intent": "IntentName"
+            },
+            "transcription": "7",
+            "transcriptionConfidence": 1
+        }
+    ],
+    "sessionState": {
+        "sessionAttributes": {
+            "attribute1": "1"
+        },
+        "activeContexts": [],
+        "intent": {
+            "slots": {
+                "Slot1": null,
+                "Slot2": null,
+                "Slot3": {
+                    "shape": "Scalar",
+                    "value": {
+                        "originalValue": "7",
+                        "resolvedValues": [
+                            "7"
+                        ],
+                        "interpretedValue": "7"
+                    }
+                }
+            },
+            "confirmationState": "None",
+            "name": "IntentName",
+            "state": "InProgress"
+        },
+        "originatingRequestId": "11111111-1111-1111-1111-111111111111"
+    },
+    "inputMode": "Text",
+    "bot": {
+        "aliasName": "string",
+        "aliasId": "string",
+        "name": "string",
+        "version": "DRAFT",
+        "localeId": "en_US",
+        "id": "string"
+    }
+}
```

Comment:
Ensuring the event value is assignable to the event type is an old test style. It doesn't actually test the types the way they will be used in handlers. I wouldn't bother removing this, but it's not a replacement for adding to the above tests.